cli/src/local/event_buffer.js

```diff
@@ -1,7 +1,7 @@
 /* @flow */
 
 type EventBufferMode = 'idle' | 'timeout'
-type FlushCallback<EventType> = (EventType[]) => void
+type FlushCallback<EventType> = (EventType[]) => any
```

Comment:
why this change ?

Reply:
Needed to make `LocalWatcher#handleEvents()` async and use `await` statements in it.
Otherwise `yarn flow` complains about the return type.

Reply:
It could also turn out to be useful in future tests.